example/tests/test_handler.py

```diff
@@ -4,6 +4,14 @@
 
 import pytest
 
+from aws_lambda_powertools import Tracer
+
+
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
In case its not obvious - I had to add this to avoid side effects from tests in other modules.

Reply:
yep :( I wish there was a cleaner way to keep config across instances and not requiring a fixture